app/views/accounts/transactions/_transaction.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= turbo_frame_tag dom_id(transaction), class: "grid grid-cols-12 items-center text-gray-900 py-4 text-sm font-medium px-4" do %>
+  <div class="col-span-4">
+    <%= render "transactions/name", transaction: transaction %>
+  </div>
+
+  <div class="col-span-3">
+    <%= render "transactions/categories/badge", category: transaction.category %>
+  </div>
+
+  <%= link_to transaction.account.name,
+              account_path(transaction.account),
+              class: ["col-span-3 hover:underline"] %>
+
+  <div class="col-span-2 ml-auto">
+    <%= render "transactions/amount", transaction: transaction %>
+  </div>
+<% end %>
```

Comment:
As the global transactions UI is getting a lot more complex with bulk selection and inline editing, I decided to make the transaction rows in the account view read-only for the time being.

Eventually we'll probably want to mirror some of the functionality implemented in the global view here, but trying to manage this in two spots adds unnecessary complexity right now.